js_modules/dagster-ui/packages/ui-core/src/runs/RunActionsMenu.tsx

```diff
@@ -518,6 +516,7 @@ export const RUN_ACTIONS_MENU_RUN_FRAGMENT = gql`
       repositoryName
       repositoryLocationName
     }
+    hasRunMetricsEnabled
```

Comment:
uber nit, one could call this just 'runMetricsEnabled' ('has' and 'enabled' are arguably redundant)